src/app/components/Onboard/Welcome/index.jsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import Template from "../Shared/template";
+import FAQs from "./faqs";
+import Features from "./features";
+import {
+  GlobeAltIcon,
+  LightningBoltIcon,
+  MailIcon,
+  ArrowRightIcon,
+  ScaleIcon,
+} from "@heroicons/react/outline";
+
+const features = [
```

Comment:
I would inline `features` and `faqs` variables into component definition. But not sure why. I guess as it is content and therefore belongs in the body :)

Reply:
I'd disagree on this one, more smaller components FTW. Esp, when using storybook which has been install on the project but not utilised in this PR.